actionview/lib/action_view/dependency_tracker.rb

```diff
@@ -1,9 +1,12 @@
 require 'thread_safe'
+require 'action_view/path_set'
 
 module ActionView
   class DependencyTracker # :nodoc:
     @trackers = ThreadSafe::Cache.new
 
+    mattr_accessor(:view_paths) { ActionView::PathSet.new }
```

Comment:
The only place I found that I could steal them from was `ActionController::Base`'s, which I can't depend on here. So this is a fallback.

Do you know of another place I could steal paths from? :smile: